index.bs

```diff
@@ -4663,6 +4663,10 @@ Issue: Specify this in terms of <a href="https://html.spec.whatwg.org/multipage/
 The user agent must periodically run [=queue reports for delivery=] on the
 [=event-level report cache=] and [=aggregatable attribution report cache=].
 
+Note: The user agent may choose to define certain criteria to ignore
+sending a certain report, for example if the report's time of
+existence exceeds a user agent defined length.
```

Comment:
Let's remove the `Note: ` prefix, since we want the ability to do this to be normative, and notes are not:

```suggestion
The user agent may delete reports without sending them according
to [=implementation-defined=] criteria.

<p class=note>For example, this can be used to limit the maximum age of
any report that is sent.
```